renku_notebooks/api/__init__.py

```diff
@@ -30,4 +31,36 @@ def health():
     return Response("service running under {}".format(config.SERVICE_PREFIX))
 
 
+@bp.route("/version")
+def version():
+    """
+    Return notebook services version.
+
+    ---
+    get:
+      description: Information about notebooks service.
+      responses:
+        200:
+          description: Notebooks service info.
+          content:
+            application/json:
+              schema: VersionResponse
+    """
+    from ..version import __version__
+
+    info = {
+        "name": "renku-notebooks",
+        "versions": [
+            {
+                "version": __version__,
+                "data": {
+                    "anonymous_sessions_enabled": config.ANONYMOUS_SESSIONS_ENABLED,
```

Comment:
```suggestion
                    "anonymousSessionsEnabled": config.ANONYMOUS_SESSIONS_ENABLED,
```

Reply:
This is more in line with naming in the values file. And I would like to transition to returning camelCase in endpoints as much as possible. Whereas internal python code would be snake_case.